exercises/patterns/printPlus/printPlus.js

```diff
@@ -1,52 +1,25 @@
-/*
-  Require the printHelpers module, which allows us to do things like print
-  characters without always inserting a new line, print characters multiple
-  times, etc.
-*/
 let helpers = require('../printHelpers');
 
-/**
- * Given an odd integer of `height`, prints "plus" pattern `height` characters
- * tall consisting of `#` characters.
- *
- * Note:
- * 1. The input must be an ODD integer or else printing a "plus" pattern
- * isn't possible since there's no "middle point" for the two lines to cross.
- * 2. This PRINTS a plus pattern, it does not RETURN anything.
- *
- * @example
- * printPlus(3); // Prints the following:
- *  #
- * ###
- *  #
- *
- * @example
- * printPlus(5); // Prints the following:
- *   #
- *   #
- * #####
- *   #
- *   #
- *
- * @param {number} height - The height of the plus symbol to print. It must be
- *  an odd integer.
- */
 function printPlus(height) {
   // If height is not an odd integer, throw an error.
   if (!Number.isInteger(height) || height % 2 !== 1) {
     throw new Error(`height must be an odd integer, received: ${height}`);
   }
 
   for (let i = 0; i < height; i++) {
-    // This is your job. :)
+    if (i === Math.floor(height/2)) {
+      helpers.printCountTimes('#', height);
+    }
+    else {
+      let numSpaces = Math.floor(height / 2)
```

Comment:
You can move this line to line 8, just outside the for loop, because numSpaces has to be calculated just once, not at each loop.